desafio-02/roberto0arruda/python/README.md

```diff
@@ -0,0 +1,22 @@
+# 2: PRIMOS
+
+## Listando números primos
+
+### Desafio
+
+Escrever um programa para listar todos os números primos entre 1 e 10000, na linguagem de sua preferência.
+
+### Solução
+
+``` python
+for num in range(1, 10000 + 1):
+    tot = 0
+    for n in range(1, num + 1):
```

Comment:
Comece o range de 2 (já que todo mundo é divisível por 1) e vá até o numero menos 1 (já que o numero é divisivel por si mesmo). Algo tipo `for n in range(2, num)`
(veja mais abaixo)

Reply:
Muito obrigado pela dica, o código foi reduzido a 6 linhas